### PR TITLE
Modify content to align with the present

### PR DESCRIPTION
**Pull request by pipo02mix**
Revamp network policy guide included the latest additions in our clusters

### CODE REVIEW

File: src/content/guides/limiting-pod-communication-with-network-policies/index.md [hunk 1]
```diff
@@ -11,54 +11,130 @@ tags = ["tutorial"]
...
+            app: my-app-that-needs-access-to-ksm
+      ports:
+        - protocol: TCP
+          port: 10301
+```
+
+To make it more visual, this how it will look at the communication between namespaces.
 
-No matter if you set default policies or not, you can limit access to certain Pods by creating a NetworkPolicy that selects them.
+(PLACEHOLDER FOR IMAGE)
```
Comment (pipo02mix) [inline src/content/guides/limiting-pod-communication-with-network-policies/index.md]: working in the diragram

**Comment by pipo02mix** [comment]
I cannot understand the error with the URLs

```
URL        `https://kubernetes.io/docs/admin/authorization/rbac/#referring-to-subjects'
Name       `official RBAC documentation'
Parent URL http://docs:8080/guides/securing-with-rbac-and-psp/, line 364, col 87
Real URL   https://kubernetes.io/docs/admin/authorization/rbac/#referring-to-subjects
Check time 2.249 seconds
Size       7KB
Result     Error: 404 Not Found
```
But the URL is there, and try to build it again
@marians any idea?

**Comment by marians** [comment]
The upstream URL

    https://kubernetes.io/docs/admin/authorization/rbac/

changed to

    https://kubernetes.io/docs/reference/access-authn-authz/

Hence the linkchecker failures. (linkchecker does not pickl up redirects apparently)

File: src/content/guides/limiting-pod-communication-with-network-policies/index.md [hunk 2]
```diff
@@ -11,54 +11,130 @@ tags = ["tutorial"]
...
+            app: my-app-that-needs-access-to-ksm
+      ports:
+        - protocol: TCP
+          port: 10301
+```
+
+To make it more visual, this how it will look at the communication between namespaces.
 
-No matter if you set default policies or not, you can limit access to certain Pods by creating a NetworkPolicy that selects them.
+![](/img/giant-swarm-network-policies-diagram.png)
```
Comment (marians) [inline src/content/guides/limiting-pod-communication-with-network-policies/index.md]: Can you please try placing this in the same folder as `index.md` and using only the file name as a reference? In our current HUGO version that should work. (Please check locally in preview)

File: src/content/guides/limiting-pod-communication-with-network-policies/index.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 +++
 title = "Limiting Pod Communication with Network Policies"
 description = "Guide on how to limit Pod communication using Network Policies"
-date = "2017-11-10"
+date = "2019-11-12"
```
Comment (marians) [inline src/content/guides/limiting-pod-communication-with-network-policies/index.md]: That's in the future ;-)

Comment (marians) [inline src/content/guides/limiting-pod-communication-with-network-policies/index.md]:
```suggestion
date = "2019-11-07"
```

A date in the future will make this page invisible until we build again on or after that date.

File: src/content/guides/limiting-pod-communication-with-network-policies/index.md [hunk 4]
```diff
@@ -11,54 +11,130 @@ tags = ["tutorial"]
 
 You can limit communication to Pods using the Network Policy API of Kubernetes.
 
-Currently the API only includes Ingress rules, i.e. you can only limit traffic to groups of Pods. Egress rules, limiting outgoing traffic from Pods, are coming to the API soon. Until then you have to use the Egress functionality of our Network Plugin, Calico, directly.
+The Network policy functionality is implemented by a network provider thanks to the CNI plugin. Nowadays there are some provides compatible (Calico, Cilium, Kube-router, ...). Giant Swarm uses Calico as a provider so out of the box users can make use of these policies.
```
Comment (puja108) [inline src/content/guides/limiting-pod-communication-with-network-policies/index.md]:
```suggestion
The Kubernetes Network Policy functionality is implemented by different network providers , like Calico, Cilium, Kube-router, etc. Most of these providers have some added functionalties that go above the main Kubernetes Network Policy API. Giant Swarm deploys Calico as a provider so users can make use of its functionality out of the box.
```

File: src/content/guides/limiting-pod-communication-with-network-policies/index.md [hunk 5]
```diff
@@ -11,54 +11,130 @@ tags = ["tutorial"]
...
+To make it more visual, this how it will look at the communication between namespaces.
 
-No matter if you set default policies or not, you can limit access to certain Pods by creating a NetworkPolicy that selects them.
+![](giant-swarm-network-policies-diagram.png)
 
-#### Allowing specific pod to pod access
+### Allowing specific pod to pod access
 
-In the following example we allow traffic to Pods labeled `role: backend` from Pods with the `role: frontend` label and only on TCP port 6379.
+In the following example, we allow traffic to Pods labelled `role: backend` from Pods with the `role: frontend` label and only on TCP port 6379.
```
Comment (rossf7) [inline src/content/guides/limiting-pod-communication-with-network-policies/index.md]:
```suggestion
In the following example, we allow traffic to Pods labeled `role: backend` from Pods with the `role: frontend` label and only on TCP port 6379.
```

Both spellings are correct but labeled is preferred in 🇺🇸 which is what we aim for. https://www.grammarly.com/blog/labeled-labelled/

File: src/content/guides/limiting-pod-communication-with-network-policies/index.md [hunk 6]
```diff
@@ -104,7 +180,7 @@ spec:
             name: freeforall
 ```
 
-Note that the namespace you apply this policy to needs to carry a label `name:` similar to the actual name key in its metadata:
+Note that the namespace you apply this policy needs to carry a label `name:` similar to the actual name key in its metadata:
```
Comment (rossf7) [inline src/content/guides/limiting-pod-communication-with-network-policies/index.md]:
```suggestion
Note that the namespace you apply this policy to needs to carry a label `name:` similar to the actual name key in its metadata:
```

**Comment by pipo02mix** [comment]
I am going ahead thx for reviews

**Pull request merged**